ansible_collections/arista/cvp/docs/schema/cv_change_control_v3.md

```diff
@@ -0,0 +1,23 @@
+# Schema for cv_change_control_v3
+
+| Variable | Type | Required | Default | Choices | Description |
+| -------- | ---- | -------- | ------- | ------------------ | ----------- |
+| change_id | list | No |  |  | List of change IDs to get/remove |
+| name | str | No |  |  | The name of the change control. If not provided, one will be generated |
+| state | str | No | show | show<br>set<br>remove | Set if we should get, set/update, or remove the change control |
+| change | Dict | No |  |  | A dict containing the change control to be created/modified |
+| &nbsp;&nbsp;&nbsp;&nbsp;name | str | No |  |  | Name of change control |
+| &nbsp;&nbsp;&nbsp;&nbsp;notes | str | No |  |  | Any notes that you want to add |
+| &nbsp;&nbsp;&nbsp;&nbsp;stages | Dict | Yes |  |  |  |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-name | str | Yes |  |  | Name of stage |
```

Comment:
This list example and the ones below seem to be missing spacing. For example `-name` vs `- name`